src/components/settings/common/Table.js

```diff
@@ -64,12 +79,13 @@ class Table extends React.Component {
     let tableContents = [];
 
     if (this.props.title) {
+      const { label: { main, secondary } } = this.props.title;
       const title = (
         <caption
-          key={this.props.title.label}
+          key={main}
           className={this.props.title.className}
```

Comment:
so when looking at the whole file just a few lines above 

```
const { label: { main, secondary } } = this.props.title;
```

as we are already pulling stuff from the title prop would it not make sense to also get the `className` at the same time ?